index.html

```diff
@@ -2905,6 +2915,166 @@ <h3>Evidence</h3>
 
       </section>
 
+      <section>
+        <h3>Confirmation Method</h3>
+
+        <p>
+A Confirmation Method can be included by an <a>issuer</a> in a <a>verifiable
+credential</a> to declare that the <a>subject</a> controls, or a <a>holder</a>
```

Comment:
I advise to rephrase this from "the subject" to "a subject". Verifiable Credentials do not have "the" subject, as there is the possibility to have multiple credentialSubjects in there. Especially if VCs might explain relationsships, it makes sense to have confirmationMethods in multiple, different credential subjects.

Reply:
```suggestion
credential</a> to declare that a <a>subject</a> controls, or a <a>holder</a>
```